goamiddleware/jwt_token_context_whitebox_test.go

```diff
@@ -68,8 +69,34 @@ func (s *TestJWTokenContextSuite) TestHandler() {
 	require.Error(s.T(), err)
 	assert.Equal(s.T(), "next-handler-error", err.Error())
 	header := textproto.MIMEHeader(rw.Header())
-	assert.NotContains(s.T(), header, "WWW-Authenticate")
-	assert.NotContains(s.T(), header, "Access-Control-Expose-Headers")
+	require.Empty(s.T(), header.Get("WWW-Authenticate"))
```

Comment:
I wonder why this test pass. Do we register the token in GenerateServiceAccountToken()?